package.json

```diff
@@ -38,7 +38,7 @@
   "files": [
     "neptune-for-graphql.mjs",
     "./templates/CDKTemplate.js",
-    "./templates/JSResolverOCHTTPSTemplate.js",
```

Comment:
Have you tested this change by executing the utility to create an app sync API and checking the queries still work with the app sync UI?